### ISSUE TITLE
Error parsing lists with code snips

### ISSUE THREAD

**Issue by Ch4s3** [pragdave/earmark#50]
The following breaks:   
````
Some notes about this command:
   + ```_4.1.9_``` specifies the version of Rails in case you have multiple versions installed.
   + ```--database=postgresql``` sets the database as Postgres
   + There is no need to run ```Jruby -S``` before ```rails new``` since you're using RVM.
````
And shows up as:
<img width="687" alt="screen shot 2015-11-24 at 12 21 50 am" src="https://cloud.githubusercontent.com/assets/1943540/11358538/6a964b44-9241-11e5-9972-dabb8e1d987f.png">

**edit to show actual markdown, as I forgot I was typing into a markdown editor

**Comment by RobertDober** [comment]
Seems you want to reopen this issue for the correct project, this is *Earmark* not Rails.

**Comment by Ch4s3** [comment]
No, I use Earmark for a blog backed by Elixir. I just noticed this while moving over an article from my old blog that just happens to be about JRuby and Rails. I did intend to post here.

Here's the page view:

```elixir
defmodule CodeBlog.PageView do
  use CodeBlog.Web, :view
  alias Earmark.Options

  def markdown(body) do
    body
    |> Earmark.to_html(%Options{gfm: true})
    |> raw
  end
end
```

and relevant partial:
```elixir
<%= for post <- @posts do %>
  <div class="post-title">
    <h2><%= post.title %></h2>
  </div>
  <div class="post-body">
    <%= markdown(post.body) %>
  </div>
<% end %>
```

**Comment by RobertDober** [comment]
Please give us something to work with. Which markdown input breaks what and how?
I cannot reopen the issue without that information.

**Comment by Ch4s3** [comment]
Oh sorry, I didn't notice that it didn't escape.
````
   + ```_4.1.9_``` specifies the version of Rails in case you have multiple versions installed.
   + ```--database=postgresql``` sets the database as Postgres
   + There is no need to run ```Jruby -S``` before ```rails new``` since you're using RVM.
````
This should be exactly what I used. I think the problem is with the +'s for the list. Here's a [link](https://gist.githubusercontent.com/Ch4s3/66ff107f92a05a29b4ac/raw/226e6832aea8e5a006aa50e8a1d26ef28b526bdb/jruby_rails.md) to a raw gist of the whole doc.

**Comment by RobertDober** [comment]
No problem I might have guessed. I was not very inspired. 
In the meantime, in case you have control over the input you could replace the triple backticks with single ones, but I guess you can't.
I'll have a look as soon as possible.

**Comment by Ch4s3** [comment]
I can definitely do that in the meantime. I'll see if I can figure out the issue, but I'm still pretty new to elixir, no promises.

**Comment by RobertDober** [comment]
Dave's code is worth studying by all means. I am about to fix this with #48 (as I will probably treat pairs of ``` that are not fences and pairs of ` equally) so it is a little bit unlikely that I will accept a PR for this one. But that said I do not want you to discourage to make one, I just warn you not to be dissapointed as I am working on this now.

Anyway the issue report is appreciated, even if I did not get it immediately ;).

**Comment by Ch4s3** [comment]
Cool. I'll poke around the code over the holiday and maybe look for other opportunities. Yeah, Dave's work is pretty amazing.

**Comment by RobertDober** [comment]
closed with 55828d1131581554712849bbacd82a88568ba66c
